CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing to OS2iot
+
+Thank you for contributing!
+
+We understand that creating a public IoT standard like this, can only be set in collaboration with as many public technologists, policy makers and interested folk as possible. Thus we appreciate your input, enjoy feedback and welcome improvements to this project and are very open to collaboration.
+
+We love issues and pull requests from everyone. If you're not comfortable with GitHub, you can use our community to give your feedback at www.os2iot.os2.eu
```

Comment:
Url is wrong - Should be https://os2iot.os2.eu - using www. in front of subdomain will result in 404 not found.